README.md

```diff
@@ -1,26 +1,28 @@
 # Videoroom
 
 Videoroom is an open-source, basic video conferencing platform using WebRTC.
-It is based on [jellyfish](https://github.com/jellyfish-dev/jellyfish), a general-purpose media server. 
+It is based on [jellyfish](https://github.com/jellyfish-dev/jellyfish), a general-purpose media server.
 Videoroom may be a good starting point for building your own real-time communication solution using Elixir and Jellyfish.
 
-## Installation 
+## Installation
 
 Make sure to have installed [Elixir](https://elixir-lang.org/install.html) first.
 
 Running the Videoroom requires connecting to an instance of [Jellyfish Server](https://github.com/jellyfish-dev/jellyfish).
 
 When running locally, you can start an instance of Jellyfish inside docker using docker compose.
+
 ```
 INTEGRATED_TURN_IP=<your ip in local network> docker compose -f docker-compose-dev.yaml up
 ```
 
 Now you can start the Videoroom:
-  * Run `mix setup` to install and setup dependencies
-  * Start Phoenix server with `mix phx.server`
 
+- Run `mix setup` to install and setup dependencies
+- Start Phoenix server with `mix phx.server`
 
 When running in production you must specify the address of the Jellyfish, as well as the authentication token via the environment variables:
```

Comment:
I think we should try to change this fragment (e.g. rephrase this sentence not to use the word 'production', or perhaps move it further down), as it could potentially be confusing -- when reading from top to bottom, you first encounter this particular sentence, and then an entire paragraph 'Production', which doesn't mention these specific `JELLYFISH_` env variables at all. If someone were to jump to the 'Production' paragraph, they would miss the mention of these variables entirely...

Reply:
I rephrased a little.